src/main/webapp/app/config/constants.tsx

```diff
@@ -124,15 +124,61 @@ export const LEVEL_BUTTON_DESCRIPTION = {
   '4': EVIDENCE_TYPE.BIOLOGICAL_EVIDENCE,
   R1: EVIDENCE_TYPE.STANDARD_CARE,
   R2: EVIDENCE_TYPE.CLINICAL_EVIDENCE,
+  Dx1: 'Dx1 description',
+  Dx2: 'Dx2 description',
+  Dx3: 'Dx3 description',
+  Px1: 'Px1 description',
+  Px2: 'Px2 description',
+  Px3: 'Px3 description',
+};
+
+export const LEVEL_TYPES = ['Tx', 'Dx', 'Px'];
+export const LEVEL_TYPE_NAMES = {
```

Comment:
So once you have the enum above, you can use the enum as the key of this object.